README.md

```diff
@@ -125,11 +153,12 @@ Sets up environment and rules for packaging `python` packages.
 Sets up the environment for building `sphinx` documentation and provides the necessary targets.
 
 #### `mfZynq.mk`
-Extra definitions for building on a Xilinx `Zynq` SoC
+Extra definitions for building on a Xilinx `Zynq` SoC (currently specific to the UW CTP7)
 
+* `PETA_STAGE` is the location where the compiler will look for headers and libraries when cross-compiling, defaults to `$(PETA_PATH)`, may be overridden if developing features in multiple projects simultaneously.
```

Comment:
How should `PETA_STAGE` be overridden when working with multiple backends?

Reply:
In an appropriate future update to `mfZynq.mk`, the flags and `INSTALL_PATH` would be wrapped inside target specific conditionals
```diff
diff --git a/mfZynq.mk b/mfZynq.mk
index 286d030..ddefba7 100644
--- a/mfZynq.mk
+++ b/mfZynq.mk
@@ -1,13 +1,13 @@
-PETA_STAGE?=$(PETA_PATH)/$(TARGET_BOARD)
+PETA_STAGE?=$(PETA_PATH)

 CFLAGS= -fomit-frame-pointer -pipe -fno-common -fno-builtin \
        -Wall -std=c++14 \
        -march=armv7-a -mfpu=neon -mfloat-abi=hard \
        -mthumb-interwork -mtune=cortex-a9 \
        -DEMBED -Dlinux -D__linux__ -Dunix -fPIC \
-       --sysroot=$(PETA_STAGE)
+       --sysroot=$(PETA_STAGE)/$(TARGET_BOARD)

-LDFLAGS+=--sysroot=$(PETA_STAGE)
+LDFLAGS+=--sysroot=$(PETA_STAGE)/$(TARGET_BOARD)

 INSTALL_PATH=/mnt/persistent/$(ShortProject)
```

Thus, the only developer change would be to point `PETA_STAGE` to the location where the compile time dependencies would live.

Reply:
Ok, maybe in a future update. Currently, I have to set `PETA_PATH` (i.e., `PETA_STAGE?=$(PETA_PATH)/$(TARGET_BOARD)`) to get the desired behavior.